src/passport-saml/saml.ts

```diff
@@ -834,7 +847,6 @@ class SAML {
         if (decryptedAssertions.length != 1) throw new Error("Invalid EncryptedAssertion content");
 
         if (
-          this.options.cert &&
           !validSignature &&
           !this.validateSignature(decryptedXml, decryptedAssertions[0], certs!)
```

Comment:
You have changed `certs!` to `certs` i.e. removed `!` character at lines 
https://github.com/node-saml/passport-saml/blob/1c169db06027435167dc0beb4ebde38e2d49195c/src/passport-saml/saml.ts#L799
and
https://github.com/node-saml/passport-saml/blob/1c169db06027435167dc0beb4ebde38e2d49195c/src/passport-saml/saml.ts#L819
so it seems odd that this decrypted assertion signature checking case still has `..., certs!)`.

btw. if you change this certs parameter consider surrounding throw statement with `{  }` (like other similar checks even though that change would be related to code formatting and not to issue that this pull request is addressing (but this pull request's content seems to be already more than just changing cert to truly mandatory configuration option).